examples/data/exercise-metrics.php

```diff
@@ -21,7 +21,7 @@
     print("list-breakdown-values OK ✅\n");
 
     // ========== get-overall-values ==========
-    $overall = $metricsApi->getOverallValues("video_startup_time", ["timeframe" => ["7:days"]]);
```

Comment:
Unfortunately this will break our regression and release integration testing as they rely on views generated in the last 7 days...

Maybe you could put your new example instead as a comment above this `getOverallValues` call?

Reply:
Ahh, that makes sense. How about adding it below like so: